### ISSUE TITLE
Usability: NVDA is not reciting the "Name" and "shortcut key" for  the menu options present  in the visual studio code appropriately.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17224]
VS Code Version: 1.6.1
OS Version:1607
**Steps to Reproduce:**
Prerequisite: NVDA screen reader   need to open.
1.Open visual studio code.
2.Using "Alt + F", key navigate to the "File" menu option.

**Expected Result:**
NVDA should announce the "Name" and the "Shortcut key" for the menu options in the visual studio code.
Ex: NVDA should announce File Menu option Alt+F

**Actual Result:**
NVDA is not announcing the menu options appropriately.
NVDA is announcing "menu sub menu"
Note: Same issue is observed for other menu options(Edit, View, Go, Help)

**User Impact:**
Screen Reader user doesn't get to know that the options corresponds to which Menu bar.

![inspectinformation_visualstudiocodeeditor_filemenuoption](https://cloud.githubusercontent.com/assets/22956130/21212258/f9990f3e-c2b0-11e6-9d39-67f13f1e0f27.jpg)
![nvda_speechviewerinformation_vscodeeditor_filemenuoption](https://cloud.githubusercontent.com/assets/22956130/21212263/0175f316-c2b1-11e6-849b-2830ac2cd1b5.jpg)

**Comment by sbatten** [comment]
@bpasero once we resolve the issue of focusing the menubar items themselves, this should work

**Comment by sbatten** [comment]
resolved via #53924
